pkg/schemas/connection_gen.go

```diff
@@ -36,6 +36,7 @@ var ShowConnectionSchema = map[string]*schema.Schema{
 	"is_primary": {
 		Type:     schema.TypeBool,
 		Computed: true,
+		ForceNew: true,
```

Comment:
Is it necessary? ForceNew behavior is already handled in the resources (primary and secondary connection), right?

Reply:
yup, will be removed